### ISSUE TITLE
Passenger Mount Checkboxes please

### ISSUE THREAD

**Issue by PirateKain** [xod-wow/LiteMount#282]
Within the Mounts list with those rows of check boxes "Aquatic, Ground, Flying and Skyriding could you add another row of check Boxes for Passenger Mounts. This way we can set the ground passenger mounts and flying passenger mounts as we see fit. Otherwise any macro just summons any passenger mount and that is kind of useless as it'll keep summoning the wrong mount. It likes to summon mostly ground mounts in area's where you can fly and that is so time consuming waiting for it to decide to cast a flying one.

That way we can check "Aquatic, Ground, Flying and or  Skyriding and also click the checkbox for Passenger if that mount has it.
As there are currently 6 passenger ground mounts, 6 passenger flying mounts and at least 10 ride along mounts. based on what I currently own anyway. I'm sure there are more and there are more on the way.

Thank you

**Comment by mbattersby** [comment]
I really have no idea what you are intending, sorry.

**Comment by PirateKain** [comment]
???

This mount list add a fifth row for Passenger Mounts.
![2024-08-19](https://github.com/user-attachments/assets/2a96f460-8d74-4f31-bda2-9f46bb50fba4)

**Comment by PirateKain** [comment]
Here like this
![2024-08-19 (1)](https://github.com/user-attachments/assets/7147b1ff-2561-4123-ae9e-c97b294cdfd5)

**Comment by mbattersby** [comment]
I understand that. It's what that should do I have no idea about.,

What happens after you check that button? There is no in-game situation for passenger mounts. The other check boxes clearly align with situations where that is the best kind of mount,

LiteMount doesn't do anything for passenger mounts. You can make a rule for it it you like, it is in the FAQ: https://github.com/xod-wow/LiteMount/wiki/Passenger-Mounts-When-Grouped

> Otherwise any macro just summons any passenger mount and that is kind of useless as it'll keep summoning the wrong mount. It likes to summon mostly ground mounts in area's where you can fly and that is so time consuming waiting for it to decide to cast a flying one.

None of that makes sense to me, sorry. I have no idea what you are doing when you are clicking endlessly hoping to get a passenger mount.

TL;DR is I'm definitely not going to add a checkbox, but I'm trying to understand what exactly you are trying to do or are getting wrong so I can help you make it work.

**Comment by PirateKain** [comment]
There are other mount addons that summon Passenger mounts in some way or another.  BeStride Mounts, Mounty#TLV to name a few.  I'm Just liking Light Mount and was hoping you could add it like those other mount Addons.

But if you won't then I'll have to make due.

Here is one of them, except they are missing half of the mounts. I put in a request for them to be added but nothing so far. Its why I checked out LiteMount.
![Screenshot 2024-08-19 212829](https://github.com/user-attachments/assets/4339dc2e-a7d2-4314-82e1-a9f59828360b)

**Comment by PirateKain** [comment]
I did get that macro to work but sometimes it summons a ground passenger mount in flying zones and flying passenger mount in ground only zones. That is the part I'm having issue with.

**Comment by mbattersby** [comment]
If you go into the rule you just created and change the action from Mount to Smart Priority Mount it should do what you are after.
